test/tests/ver-1c/comparison_ver-1c.py

```diff
@@ -3,41 +3,99 @@
 import numpy as np
 from matplotlib import gridspec
 import pandas as pd
-from    scipy import special
+from scipy.special import erf
+from numpy import sqrt
 
-#========= Comparison of concentration as a function of time ===================
+tmap_sol = pd.read_csv("ver-1c_csv.csv")
+tmap_time = tmap_sol['time'][1:]
+tmap_conc0 = tmap_sol['point0'][1:]
+tmap_conc0_25 = tmap_sol['point0.25'][1:]
+tmap_conc10 = tmap_sol['point10'][1:]
+tmap_conc12 = tmap_sol['point12'][1:]
+
+def get_c_over_time_TMAP4(x,t):
```

Comment:
```suggestion
def get_c_analytical(x,t):
```

Reply:
I don't think we should have `TMAP8` in the name since it is the same for TMAP7, AND this is actually the corrected expression. 
Plus, it also provides C over the distance, so it's more general than just over time.